site/en/docs/extensions/whatsnew/index.md

```diff
@@ -16,6 +16,15 @@ updated: 2022-02-17
 Check this page often to learn about changes to the Chrome extensions platform, its documentation,
 and related policy or other changes.
 
+### Chrome 102: {: #m102-registercontentscripts-main-world }
+
+April 14, 2022
+
+Dynamically registered content scripts can now specify the
```

Comment:
Is it possible to declare a main world content script in the manifest? According to the available schema there is no such option: https://json.schemastore.org/chrome-manifest